kong/router/traditional.lua

```diff
@@ -1448,6 +1449,17 @@ function _M.new(routes, cache, cache_neg)
   local match_sources        = not isempty(plain_indexes.sources)
   local match_destinations   = not isempty(plain_indexes.destinations)
 
+  -- warning about the regex cache size being too small
+  if worker_id() == 0 and match_regex_uris
+    and regex_uris[0] > (kong.configuration.nginx_http_lua_regex_cache_max_entries or 1024) then
```

Comment:
should use a var to cache `kong.configuration.nginx_http_lua_regex_cache_max_entries or 1024`

Reply:
done, thanks